sdk/identity/identity/src/credentials/clientCertificateCredential.ts

```diff
@@ -105,7 +110,8 @@ export class ClientCertificateCredential implements TokenCredential {
       const header: jws.Header = {
         typ: "JWT",
         alg: "RS256",
-        x5t: this.certificateX5t
+        x5t: this.certificateX5t,
+        x5c: this.certificateX5c
```

Comment:
Shouldn't this header be conditionally added only in the case that includeX5c was specified in the options?

Reply:
If `certificateX5c` is undefined, this would set the underlying field to undefined.

Reply:
Forgive my JS ignorance here, but if the field is undefined does that mean that the key will be excluded from the serialized json?

Reply:
Ahh good point. I'll fix that.

Reply:
For posterity, the answer is yes:

![image](https://user-images.githubusercontent.com/639216/91367524-8dac4780-e7bb-11ea-9578-ecc7ad7cfe9c.png)